source/EvoPlanet/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You can leave it now, but nvm (if I understand correctly) will be used for angular version migration. However, this is not in scope for the PR. Try to limit the changes of a PR to those things in scope, or things get incoherent and harder to review.